Gulpfile.js

```diff
@@ -98,6 +99,13 @@ gulp.task('standard', () => {
     }))
 })
 
+// Run HTML Hint checks
+gulp.task('html-hint', () => {
```

Comment:
The default `gulp test` runs both **StandardJS** and the unit tests. I think we should keep the convention that `gulp test` runs _all_ tests, and maybe we need to think about a new gulp task for when you just want to run unit tests.

So I think we need to add running `htmhint` to the `test` task